ipc/provider/src/manager/evm/manager.rs

```diff
@@ -67,7 +67,7 @@ const ETH_PROVIDER_POLLING_TIME: Duration = Duration::from_secs(1);
 const TRANSACTION_RECEIPT_RETRIES: usize = 200;
 
 /// The majority vote percentage for checkpoint submission when creating a subnet.
-const SUBNET_MAJORITY_PERCENTAGE: u8 = 60;
+const SUBNET_MAJORITY_PERCENTAGE: u8 = 67;
```

Comment:
In contract [this default](https://github.com/consensus-shipyard/ipc/blob/main/contracts/test/invariants/handlers/SubnetRegistryHandler.sol#L24) is set as 70. Would it make sense to unify it one way or another so it's at least simpler to answer the question "what is the default?"

Reply:
From Byzantine literature it has to be `n * 2/3 + 1`, which if we go from 1 to 100 is 67. Not sure where 60 came from, or why it should be 70.